src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -107,14 +110,162 @@ function TokenIcons({
   )
 }
 
-function Card({ children, testID }: { children: React.ReactNode; testID: string }) {
+function Card({
+  children,
+  testID,
+  cardStyle,
+}: {
+  children: React.ReactNode
+  testID: string
+  cardStyle?: ViewStyle
+}) {
   return (
-    <View testID={testID} style={styles.card}>
+    <View testID={testID} style={[styles.card, cardStyle]}>
       {children}
     </View>
   )
 }
 
+function EarningItemLineItem({ earnItem }: { earnItem: EarningItem }) {
+  const { t } = useTranslation()
+  const tokenInfo = useTokenInfo(earnItem.tokenId)
+  const amountInUsd = tokenInfo?.priceUsd?.multipliedBy(earnItem.amount)
+  const localCurrencyExchangeRate = useSelector(usdToLocalCurrencyRateSelector)
+  const amountInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+    amountInUsd ?? 0
+  )
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+
+  return (
+    <View testID="EarningItemLineItem" style={styles.cardLineContainer}>
+      <View style={styles.cardLineLabel}>
+        <Text numberOfLines={1} style={styles.depositAndEarningsCardLabelText}>
+          {earnItem.label}
+        </Text>
+      </View>
+      <View style={styles.flexShrink}>
+        <Text style={styles.depositAndEarningsCardValueText}>
+          {t('earnFlow.poolInfoScreen.lineItemAmountDisplay', {
+            localCurrencySymbol,
+            localCurrencyAmount: formatValueToDisplay(amountInLocalCurrency),
+            cryptoAmount: formatValueToDisplay(new BigNumber(earnItem.amount)),
+            cryptoSymbol: tokenInfo?.symbol,
+          })}
+        </Text>
+      </View>
+    </View>
+  )
+}
+
+function DepositAndEarningsCard({ earnPosition }: { earnPosition: EarnPosition }) {
+  const { t } = useTranslation()
+  const { balance } = earnPosition
+  const { earningItems, depositTokenId, cantSeparateCompoundedInterest } = earnPosition.dataProps
+  const tokenInfo = useTokenInfo(depositTokenId)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const localCurrencyExchangeRate = useSelector(usdToLocalCurrencyRateSelector)
+
+  // Deposit items used to calculate the total balance and total deposited
+  const depositBalanceInUsd = tokenInfo?.priceUsd?.multipliedBy(balance)
+  const depositBalanceInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+    depositBalanceInUsd ?? 0
+  )
+
+  // Earning items used to calculate the total balance and total deposited
+  const earningItemsTokenIds = earningItems.map((item) => item.tokenId)
+  const earningItemsTokenInfo = useTokensInfo(earningItemsTokenIds)
+
+  const totalBalanceInLocalCurrency = depositBalanceInLocalCurrency.plus(
+    earningItems.reduce((acc, item) => {
+      if (item.includedInPoolBalance) return acc
```

Comment:
nit: consider using filter + reduce to make this more readable, my first assumption was the two reduces were doing the same summation. Also could make this more DRY by making a small function to sum a list of earningItems

Reply:
Used filter and reduce in 76f8603dbc40cde827d8bbf40538dd9900cc5fa2!